unity-ggjj/Assets/Editor/Ink/Validation.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Linq;
+using Ink.UnityIntegration;
+using UnityEditor;
+using UnityEngine;
+
+namespace Editor.Ink
+{
+    public class Validation : AssetPostprocessor
+    {
+        /// <summary>
+        /// Called when an Ink file is compiled after editing it
+        /// </summary>
+        private static void OnPostprocessAllAssets (string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths) {
+            foreach (var importedAssetPath in importedAssets) {
+                if (InkEditorUtils.IsInkFile(importedAssetPath))
+                {
+                    InkFile inkFile = InkLibrary.GetInkFileWithPath(importedAssetPath);
+                    ValidateNarrativeScript(inkFile);
+                }
```

Comment:
Nit: 
![image](https://user-images.githubusercontent.com/1689033/153883659-461fad64-c6d6-43ea-a0e0-7971e565f3f3.png)
: D
```suggestion
                if (!InkEditorUtils.IsInkFile(importedAssetPath))
                {
                    continue;
                }
                InkFile inkFile = InkLibrary.GetInkFileWithPath(importedAssetPath);
                ValidateNarrativeScript(inkFile);
```
It doesn't make much of a difference at the moment, but since we're already at 3 levels of indentation without the `if`-conditional, I like limiting any more to simple flow instructions.